load-tests/k6-script.js

```diff
@@ -3,7 +3,7 @@ import http from 'k6/http';
 export default function() {
   var hostname = __ENV.LOAD_TEST_HOSTNAME || "localhost:8000"
   var server_list = [hostname]
-  var endpoint_list = ["/", "/io_task", "/cpu_task", "/random_sleep", "/random_status", "/chain", "/error_test"]
+  var endpoint_list = ["/", "/io_task", "/cpu_task", "/random_sleep", "/random_status", "/chain", "/missing_page_test"]
```

Comment:
The "/error_test" endpoint doesn't exist, so this should garner an HTTP 404. I adjusted the request to make that clearer.